src/electionguard_verify/verify.py

```diff
@@ -17,13 +21,23 @@ class Verification:
     message: Optional[str]
 
 
-def verify_ballot(ballot: CiphertextBallot) -> Verification:
+def verify_ballot(
+    ballot: CiphertextBallot,
+    internal_manifest: InternalManifest,
```

Comment:
In this particular case, we do not want to send the `InternalManifest` but the `Manifest` for the method signature. 

Luckily one can move between the two. But this method should instead take a `Manifest` as a parameter.